examples/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java

```diff
@@ -395,11 +380,15 @@ public void onRoutesChanged(@NotNull List<? extends DirectionsRoute> routes) {
       for (DirectionsRoute route : routes) {
         routeLines.add(new RouteLine(route, null));
       }
-
+      long start = System.currentTimeMillis();
       mapboxRouteLineApi.setRoutes(routeLines, new MapboxNavigationConsumer<Expected<RouteSetValue, RouteLineError>>() {
         @Override
         public void accept(Expected<RouteSetValue, RouteLineError> routeDrawData) {
+          Log.e("foobar", "total calc time: " + (System.currentTimeMillis() - start));
```

Comment:
Could you share some _before_ vs _after_ numbers gains?

Reply:
Can I refer you to this: https://github.com/mapbox/mapbox-navigation-android/issues/4096#issuecomment-819904779? Although it's verbose it shows the numbers I was seeing.